docs/tutorial/t-setup-environment.md

```diff
@@ -0,0 +1,45 @@
+# Environment Setup
```

Comment:
I'm not sure of the value of having a tutorial for this charm. IMO a simply Usage section would be useful. Here we mostly have a tutorial on how to get started with Juju and that's probably not what we want to have on every single charm on Charmhub.

Reply:
t-setup-environment.md file is removed.